crawler.js

```diff
@@ -7,10 +7,21 @@ const tldts = require('tldts');
 const MAX_LOAD_TIME = 30000;//ms
 const MAX_TOTAL_TIME = MAX_LOAD_TIME * 2;//ms
 const EXECUTION_WAIT_TIME = 2500;//ms
-const VIEWPORT = {
+
+const DEFAULT_USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36';
+const MOBILE_USER_AGENT = 'Mozilla/5.0 (Linux; Android 10; Pixel 2 XL) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/82.0.4062.3 Mobile Safari/537.36';
```

Comment:
v82 seems to be a dev version - https://www.chromestatus.com/features/schedule . I'd use stable one (80) to try to blend into the crowd.